app/api.py

```diff
@@ -462,6 +462,65 @@ def update_ticket_status(ticket_id: int, status: TicketStatus):
             raise HTTPException(status_code=404, detail="Not found")
 
 
+@api_v1_router.get("/get-data")
```

Comment:
I didn't spot it during my first review, but `/get-data` is not a very informative endpoint name. `/stats` would be more appropriate.
The function name should also be modified.